go.mod

```diff
@@ -1,24 +1,28 @@
 module github.com/k8snetworkplumbingwg/sriov-cni
 
-go 1.17
+go 1.19
 
```

Comment:
you have pr #228 open for this, can you clean up this pr first before we merge ?

Reply:
So can #228 it be merged in first?